src/System.Management/tests/System/Management/ManagementObjectTests.cs

```diff
@@ -50,5 +49,31 @@ public void Set_Property_Win32_ComputerSystem()
                 obj.SetPropertyValue("Workgroup", "WmiTests");
             }
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [OuterLoop]
+        public void Invoke_Instance_And_Static_Method_Win32_Process()
+        {
+            var processClass = new ManagementClass("Win32_Process");
+            object[] methodArgs = { "notepad.exe", null, null, 0 };
+
+            object resultObj = processClass.InvokeMethod("Create", methodArgs);
+
+            Thread.Sleep(1000);
+            var resultCode = (uint)resultObj;
+            Assert.Equal(0u, resultCode);
+
+            var processId = (uint)methodArgs[3];
+            Assert.True(0u != processId, $"Unexpected process ID: {processId}");
+
+            var process = new ManagementObject($"Win32_Process.Handle=\"{processId}\"");
+            resultObj = process.InvokeMethod("Terminate", new object[]{ 0 });
+            resultCode = (uint)resultObj;
+            Assert.Equal(0u, resultCode);
+
+            Thread.Sleep(2000);
```

Comment:
even more horrible, it's assuming it terminates in no more than 2 seconds. Instead, use the Process class to wait for exit, then verify the process.Get() command here.

Reply:
Same above